File: src/main/java/dragonsaga/ui/RaceChangeCosmeticsForm.java
package dragonsaga.ui;

import necesse.entity.mobs.PlayerMob;
import necesse.gfx.forms.Form;
import necesse.gfx.forms.components.FormPlayerIcon;

public class RaceChangeCosmeticsForm extends Form {
    public final FormPlayerIcon icon;
    private PlayerMob _player;

    public RaceChangeCosmeticsForm(int x, int y, int width, PlayerMob player) {
        super(width, 0);
        this.setPosition(x, y);
        this.drawBase = false;
        _player = player;

        this.icon = this.addComponent(new FormPlayerIcon(8, 4, 128, 128, _player));
    }
}

File: src/main/kotlin/dragonsaga/ui/RaceChangeCosmeticsForm.kt
package dragonsaga.ui

import necesse.entity.mobs.PlayerMob
import necesse.gfx.forms.Form
import necesse.gfx.forms.components.FormPlayerIcon

class RaceChangeCosmeticsForm(x: Int, y: Int, width: Int, player: PlayerMob) : Form(width, 0) {
    val icon: FormPlayerIcon
    private val _player: PlayerMob

    init {
        this.setPosition(x, y)
        drawBase = false
        _player = player
        icon = this.addComponent(FormPlayerIcon(8, 4, 128, 128, _player))
    }
}